Header header
geometry_msgs/Vector3 position_delta
geometry_msgs/Vector3 angle_delta
float32 confidence
